CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 ## Unreleased
 
+## 1.0.2 2021-09-27
```

Comment:
If `artifactPrefix` and `artifactSuffix` become mandatory, isn't this technically a breaking change ? As in, if someone is currently using the generator with a given config file, and they upgrade to this version, their build could fail, right ?

Reply:
Yeah, I thought about that - but given we haven't announced it's release yet, I can't see how that would actually be a problem for anyone.
I actually also wanted to change the names of these fields to be `artifactNamePrefix` and `artifactNameSuffix` (just to more explicitly convene their intent), but I thought that risked too much of a change (but given we haven't announced it's release yet(!), what do you think, should I change them too...??)

Reply:
I have no strong opinion on the field name, other than if we want to change it it should be before making the public announcement :). But I agree with you that either way, it's a change of the same "level" as the current one (so patch here)